src/Checkbox.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Icon from './Icon'
+import Flex from './Flex'
+import Box from './Box'
+import theme from './theme'
```

Comment:
If you like single `import` statement for your imported components, maybe you can try out a single import with components in alphabetical order in title case (i.e. `Box`), and then other artifacts such as `theme` with lowercase? (the order is totally subjective opinion, but just thought to share to at least have the components alphabetically sorted in the import:
```
import { Box, Flex, Icon, theme } from '../src'
```

Reply:
*derp*